telemetry_components/src/main/java/com/newrelic/telemetry/SimpleSpanBatchSender.java

```diff
@@ -0,0 +1,32 @@
+package com.newrelic.telemetry;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.newrelic.telemetry.http.HttpPoster;
+import com.newrelic.telemetry.spans.SpanBatchSender;
+import com.newrelic.telemetry.spans.SpanBatchSenderBuilder;
+import java.time.Duration;
+
+public class SimpleSpanBatchSender {
+
+  public static SpanBatchSender build(String apiKey) {
+    return build(apiKey, Duration.ZERO);
```

Comment:
I don't think ZERO is a good value here. We should choose a good default. ZERO means infinite timeout, and we've gotten feedback that this is a bad choice.

Reply:
Heh, yeah, I was borrowing heavily from the other simple sender.  I agree tho and will change it to 2 seconds as a reasonable starting point.